hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -179,12 +180,21 @@
       tests: ["keymgr_sync_async_fault_cross"]
     }
     {
-      name: sec_cm_check
+      name: stress_all
+      desc: '''
+            - Combine above sequences in one test to run sequentially, except csr sequence and
+              keymgr_cfg_regwen (requires zero_delays).
+            - Randomly add reset between each sequence.'''
+      milestone: V2
+      tests: ["keymgr_stress_all"]
+    }
+    {
+      name: sec_cm_additional_check
       desc: '''
-            Verify the outcome after injecting faults into security countermeasures.
+            Verify the outcome after injecting faults to security countermeasures.
 
             Stimulus:
-            As mentioned in `sec_cm_testplan`.
+            As mentioned in `sec_cm_count_testplan` and `sec_cm_fsm_testplan`.
```

Comment:
In the expanded testplan, all testpoints show up in one table. the actual testplan is not reflected anywhere. We should perhaps reword this to make it clearer.

Reply:
changed to prim_*_check. Thanks